ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -184,7 +184,7 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
-- as: aws-ipi-disc-priv-amd-mixarch-f9-longduration-cloud
+- as: aws-ipi-disc-priv-amd-mixarch-tp-f9-longduration-cloud
```

Comment:
dont think we need to run every 9 days, can we update to 14? tp clusters to 28?

Reply:
It was changed to 9 by @liangxia before https://github.com/openshift/release/pull/49982/files, can we change it back?

Reply:
@liangxia We have four jobs were changed to f9, they were f28 before, not sure about the background, can we change it back? They are long duration jobs for cloud team, which may take 13 hours for each job.
aws-ipi-disc-priv-amd-mixarch-f9-longduration-cloud and azure-ipi-disc-fullypriv-amd-mixarch-f9-longduration-cloud for 4.15 and 4.16

Reply:
The higher frequency are for component readiness.
How about keep it as it's now for some time ? We will revisit it after we onboard component readiness.

Reply:
> How about keep it as it's now for some time ? We will revisit it after we onboard component readiness.

Thank you @liangxia I am ok, what do you think? @sunzhaohua2

Reply:
OK for me @huali9